css/styles.css

```diff
@@ -608,9 +612,13 @@ textarea {
 
```

Comment:
- Kindly fix the buttons style to match Figma design

![image](https://user-images.githubusercontent.com/93630700/227014364-ff1afc94-f42c-49b4-8d98-3fabde0e8fdc.png)

Reply:
@BregornOriginal  the Figma design looks like ine
please be more specific about where  I should fix it.